lib/datasource/mock/mock_repository.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/foundation.dart';
+import 'package:seeds/datasource/remote/model/account_guardians_model.dart';
+import 'package:seeds/datasource/remote/model/user_recover_model.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:async/async.dart';
+
+class MockRepository {
+  static const bool _LiveMockMode = false;
+
```

Comment:
So to enable mock guardians calls I just need set this to true?

Reply:
Yes